### PR TITLE
[LPT] Disable Move Fake Quantize on shuffle channels pattern

### PR DESCRIPTION
**Pull request by ndemashov**
### Details:
 - ReverseInputChannels pattern in Concat transformation

### Tickets:
 - 79074

### CODE REVIEW

**Review by eshoguli — COMMENTED** [review]
Please run DL Bechmark

File: src/common/low_precision_transformations/src/move_fake_quantize.cpp [hunk 1]
```diff
@@ -176,6 +176,18 @@ bool MoveFakeQuantize::canBeTransformed(const TransformationContext& context, st
     if (q_dq && (convert_q->get_output_size() != 1 || layer->get_output_size() != 1)) {
         return false;
     }
+    if (is_type<opset1::Split>(concat->get_input_node_ptr(0))) {
+        bool only_split = true;
+        const size_t id = concat->get_input_node_ptr(0)->get_instance_id();
+        for (size_t i = 1; i < concat->get_input_size(); ++i) {
+            if (concat->get_input_node_ptr(i)->get_instance_id() != id) {
+                only_split = false;
```
[the remark this answers is not in the document]
Comment (ndemashov) [reply src/common/low_precision_transformations/src/move_fake_quantize.cpp]: done, thank you.

File: src/common/low_precision_transformations/src/move_fake_quantize.cpp [hunk 2]
```diff
@@ -176,6 +176,18 @@ bool MoveFakeQuantize::canBeTransformed(const TransformationContext& context, st
     if (q_dq && (convert_q->get_output_size() != 1 || layer->get_output_size() != 1)) {
         return false;
     }
+    if (is_type<opset1::Split>(concat->get_input_node_ptr(0))) {
```
[the remark this answers is not in the document]
Comment (ndemashov) [reply src/common/low_precision_transformations/src/move_fake_quantize.cpp]: done, thank you.

**Comment by eshoguli** [comment]
please, add functional tests (+ plugin, ideally)

**Review by eshoguli — APPROVED** [review]
tests will be added in separate PR

**Pull request merged**